lib/rspec/rails/example/system_example_group.rb

```diff
@@ -102,6 +103,7 @@ def driven_by(*args, &blk)
 
         before do
           @routes = ::Rails.application.routes
+          ActiveJob::Base.disable_test_adapter
```

Comment:
I'm really unsure about this, why are we doing it? We need to be able to change adapters sure, but surely the default should be the test adapter...